src/Symfony/Bundle/FrameworkBundle/DependencyInjection/FrameworkExtension.php

```diff
@@ -3021,10 +2971,6 @@ private function registerUidConfiguration(array $config, ContainerBuilder $conta
             $container->getDefinition('name_based_uuid.factory')
                 ->setArguments([$config['name_based_uuid_namespace']]);
         }
-
-        if (!class_exists(UidValueResolver::class)) {
```

Comment:
Class was added to symfony/http-kernel with #44665 (6.1), FrameworkBundle requires 6.4+